app/src/main/res/values/strings.xml

```diff
@@ -14,12 +14,12 @@
     <string name="stations_button">Configure Stations</string>
     <string name="call_button">Call Set</string>
     <string name="decline_button">Decline Set</string>
-    <string name="stream_button">Call Set to Stream</string>
+    <string name="stream_button">Change to Stream Setup</string>
     <string name="p2DQ">DQ (P2)</string>
     <string name="p1DQ">DQ (P1)</string>
     <string name="report_set_text">Report Set to Start.gg</string>
     <string name="no_ongoing">There are no ongoing sets right now!</string>
-    <string name="no_upcoming">There are no upcoming sets right now!</string>
+    <string name="no_upcoming">There are no upcoming sets right now! Sets might take some time to appear.</string>
```

Comment:
I put a longer message here explaining the preview stuff so I think we should use that.